nautobot/core/admin.py

```diff
@@ -60,3 +57,49 @@ class NautobotModelAdmin(ModelAdmin):
         models.ForeignKey: {"widget": widgets.StaticSelect2},
         models.ManyToManyField: {"widget": widgets.StaticSelect2Multiple},
     }
+
+
+class LogEntryAdmin(NautobotModelAdmin):
```

Comment:
Wonder if this should go in the `users` app instead of `core` just for symmetry with `UserAdmin`, `GroupAdmin`, `ObjectPermissionAdmin`, and `TokenAdmin`?